modules/local/dream/differential/main.nf

```diff
@@ -20,6 +20,7 @@ process DREAM_DIFFERENTIAL {
     tuple val(meta), path("*.MArrayMM.dream.rds")       , emit: rdata
     tuple val(meta), path("*.dream.model.txt")          , emit: model
     tuple val(meta), path('*.dream.contrasts_plot.png') , emit: contrasts_png
+    tuple val(meta), path('*.dream.var_plot.png')       , emit: var_png
     tuple val(meta), path("*.R_sessionInfo.log")        , emit: session_info
     tuple val(meta), path("*.normalised_counts.tsv")    , emit: normalised_counts, optional: true
```

Comment:
Nice to have: the `.yml` file that explains inputs and outputs. This way it can be ready for submission to nf-core modules.